app/javascript/packages/document-capture/context/failed-capture-attempts.jsx

```diff
@@ -44,18 +48,25 @@ FailedCaptureAttemptsContext.displayName = 'FailedCaptureAttemptsContext';
  *
  * @prop {ReactNode} children
  * @prop {number} maxFailedAttemptsBeforeTips
+ * @prop {number} maxAttemptsBeforeNativeCamera
```

Comment:
Will a user ever see these tips anymore? If not, is this a feature we'd want to remove?

Reply:
@aduth good catch -- we are currently all discussing this and will get back to you / push changes if needed.

Reply:
Ok. We have decided to open a separate ticket for dealing with the tips issue. It's going to take us a bit more time to sort that all out. As it stands with this branch, the tips should never be triggered (the default is 3 failed attempts vs our 2).